pkg/commands/describe.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 Praetorian Security, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/praetorian-inc/trident/pkg/db"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	// identifier for the campaign
+	campaignID string
+)
+
+var describeCmd = &cobra.Command{
```

Comment:
for this PR, we would like the describe and list to be subcommands of `campaign`. Currently, there is only one command that is part of the `campaign` command (create). for describe and list to be subcommands, the current `campaign` command will need to be reworked to become `campaign create` and then describe and list will need to be added as subcommands. Luckily, this will not require much refactoring, as cobra makes it easy to just shift around the vars you declare here and add them to the campaign command with AddCommand.